core/repository/block/pg.ts

```diff
@@ -144,8 +140,8 @@ class BlockPGRepo implements IBlockPGRepository {
     }
 
     async isExist(blockId: BlockId): Promise<boolean> {
-        let exists = await db.oneOrNone(queries.isExist, { blockId });
-        return exists === 't';
+        let response = await db.one(queries.isExist, { blockId });
```

Comment:
Method `one` will be created exceptions when a block will be not exists

Reply:
But I don't know how works `SELECT EXISTS`. Maybe exception will not happen

Reply:
This function always returns value